libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.scss

```diff
@@ -73,6 +73,16 @@ ion-header ion-toolbar {
   --border-width: 0;
   --background: transparent;
   --color: #{get-color('black')};
+
+  ion-title {
+    $spacingToTop: 30px;
```

Comment:
If you need a magic number (I'm not saying you do in this case 😉) it's a good point moving it into a named var. SCCS vars are not scoped though, så consider declaring var at top of file instead...

Reply:
Got it. Thank you for specificity on how variables in SCSS scope. Personally, I'd prefer to keep it there, as it is the only place we'd use it, and I'd also argue, for the same reason, that addding it to the top of the file adds extra noice to a already pretty populated modal-wrapper.component.scss. However, I sympathize with the convention of adding scss variables to top of file.